tests/units/feature_extraction/test_feature_calculations.py

```diff
@@ -1221,26 +1222,47 @@ def test_count_above(self):
         self.assertEqualPandasSeriesWrapper(count_above, [1] * 10, 1, t=1)
         self.assertEqualPandasSeriesWrapper(count_above, list(range(10)), 1, t=0)
         self.assertEqualPandasSeriesWrapper(count_above, list(range(10)), 0.5, t=5)
-        self.assertEqualPandasSeriesWrapper(count_above, [0.1, 0.2, 0.3] * 3, 2/3, t=0.2)
-        self.assertEqualPandasSeriesWrapper(count_above, [np.NaN, 0, 1] * 3, 2/3, t=0)
-        self.assertEqualPandasSeriesWrapper(count_above, [np.NINF, 0, 1] * 3, 2/3, t=0)
+        self.assertEqualPandasSeriesWrapper(count_above, [0.1, 0.2, 0.3] * 3, 2 / 3, t=0.2)
```

Comment:
I would probably not format it like this, but PEP8 does not enforce any of the two possibilities (https://stackoverflow.com/questions/18171560/does-pep-8-require-whitespace-around-operators-in-function-arguments). So I will leave it as it is if you think it is better like this!

Reply:
This change was automatically made by the IDE (PyCharm CE 2020.1)